nemo/collections/llm/api.py

```diff
@@ -937,6 +939,7 @@ def export_ckpt(
     output_path: Optional[AnyPath] = None,
     overwrite: bool = False,
     load_connector: Callable[[Path, str], io.ModelConnector] = load_connector_from_trainer_ckpt,
+    modelopt_export_kwargs: dict[str, Any] = {},
```

Comment:
Let's default to `None` and set `kwargs` to `{}` by hand. I don't quite expect this is really risky here but anyway it's a good practice to avoid mutable defaults see, for example, https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil